src/UhdmAst.cpp

```diff
@@ -860,12 +869,16 @@ namespace UhdmAst {
 
           // Get parameter assignments
           itr = vpi_iterate(vpiParamAssign, obj_h);
+          std::map<std::string, AstNode*> parameter_map;
           while (vpiHandle vpi_child_obj = vpi_scan(itr) ) {
             vpiHandle param_handle = vpi_handle(vpiLhs, vpi_child_obj);
             std::string param_name = vpi_get_str(vpiName, param_handle);
             sanitize_str(param_name);
             UINFO(3, "Got parameter (pin) " << param_name << std::endl);
             auto is_local = vpi_get(vpiLocalParam, param_handle);
+            std::string is_imported;
```

Comment:
This name is confusing. What exactly does this represent? Is it the name of the package?

Reply:
This reflects the way it is stored in UHDM: if there is no vpiImported node, module was not imported (and the string is empty), otherwise it contains either the name of the package the module was imported from.
I agree that this is confusing, since we do not need the name of the package at this point I changed this to a `bool` function.